orbit/db2.py

```diff
@@ -1,8 +1,14 @@
 import config
 from peewee import *
+from typing import Optional, Self
 
 DB = SqliteDatabase(config.database)
 
+# Helpful peewee orm docs:
+# http://docs.peewee-orm.com/en/latest/peewee/models.html
+# http://docs.peewee-orm.com/en/latest/peewee/querying.html
+
+
```

Comment:
not sure if this should stay in final version, helpful for dev maybe, but should be removed imo, or if we really want it, at least at the top of the file instead of buried partway down, or even better, in a readme.

Reply:
yea, helpful for dev, which is what comments are for :p